ferrilata-devilline.IntegrationTests/Scenarios/IntegrationTests.cs

```diff
@@ -12,5 +21,185 @@ public IntegrationTests(TestContext testContext)
         {
             _testContext = testContext;
         }
+
+        [Theory]
+        [InlineData("api/post/pitch")]
+        public async Task PostPitchCorrect_AuthorizationPresent(string url)
+        {
+            var newPosting = new Pitch
+            {
+                BadgeName = "English speaker",
+                OldLVL = 2,
+                PitchedLVL = 3,
+                PitchMessage = "Hello World! My English is bloody gorgeous.",
+                Holders = new[] { "balazs.jozsef", "benedek.vamosi", "balazs.barna" }.ToList()
+            };
+
+            var PostingJson = JsonConvert.SerializeObject(newPosting);
+
+            var client = _testContext.Client;
+
+            var request = new HttpRequestMessage(HttpMethod.Post, url);
+
+            request.Content = new StringContent(PostingJson,
+                                    Encoding.UTF8,
+                                    "application/json");
+            request.Headers.Add("Authorization", "something");
+
+            var response = await client.SendAsync(request);
+
```

Comment:
Too many unnecessary empty lines, try to separate the `arrange`, `act`, `assert` parts maybe